### PR TITLE
Test cvd load on presubmit

### PR DESCRIPTION
**Pull request by jemoreira**

### CODE REVIEW

File: e2etests/environment_specs/aosp_main_x64_phone_x2.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "instances": [
+    {
+      "name": "ins-1",
+      "disk": {
+        "default_build": "@ab\/aosp-main\/aosp_cf_x86_64_phone-trunk_staging-userdebug"
+      }
+    }, {
```
Comment (ser-io) [inline e2etests/environment_specs/aosp_main_x64_phone_x2.json:8]: nit:  new line.

File: e2etests/boot_tests.bzl [hunk 2]
```diff
@@ -10,3 +10,17 @@ def launch_cvd_boot_test(name, branch, target):
             "no-sandbox",
         ],
     )
+
+def cvd_load_boot_test(name, env_file, size = "medium"):
```
Comment (ser-io) [inline e2etests/boot_tests.bzl:14]: do we need boot in the name? why not `cvd_load_test`?

File: e2etests/environment_specs/aosp_main_x64_phone_x2.json (hunk 1 above)
Comment (jemoreira) [reply e2etests/environment_specs/aosp_main_x64_phone_x2.json]: done

File: e2etests/boot_tests.bzl (hunk 2 above)
Comment (jemoreira) [reply e2etests/boot_tests.bzl:14]: Probably not, but that naming scheme is in the other tests. If you want to see it changed I'll do it in a different PR.

The reason for the "boot" in the name is legacy: Cuttlefish is used to run CTS, so when you only want to make sure it boots it's usually called "boot test".

**Review by ser-io — APPROVED** [review]

**Pull request merged**